config.json

```diff
@@ -689,6 +689,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 7
+      },
```

Comment:
@glennj Subject for another PR, but thoughts on sorting these? By difficulty then name?